lib/house.rb

```diff
@@ -0,0 +1,62 @@
+class House
+attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    rooms << room
+  end
+
+  def above_market_average?
+    if @price[1..-1].to_i > 500000
+      true
+    else
+      false
+    end
+  end
+
+  def rooms_from_category(category)
```

Comment:
Great job using each to get the job done here! You can save a few lines with `find_all`: 

`@rooms.find_all {|room| room.category == category}`